### PR TITLE
Analytics Implementation for Resources for Schools [SCO Migration] #3055

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

For EACH individual link in 'Latest Announcement from VA' section

```
dataLayer.push({
  'event': 'nav-featured-content-link-click',
  'featured-content-header': 'Latest announcement from VA' 
});
```

For EACH individual link in 'Other Resources to support your students' section

```
dataLayer.push({
  'event': 'nav-linkslist',
  'links-list-header': 'Other resources to support your students' 
});
```

https://github.com/department-of-veterans-affairs/va.gov-team/issues/3055

## Acceptance criteria
- [ ] For EACH individual link in 'Latest Announcement from VA' section

```
dataLayer.push({
  'event': 'nav-featured-content-link-click',
  'featured-content-header': 'Latest announcement from VA' 
});
```

- [ ] For EACH individual link in 'Other Resources to support your students' section

```
dataLayer.push({
  'event': 'nav-linkslist',
  'links-list-header': 'Other resources to support your students' 
});
```

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/components/merger-crosslinks.html [hunk 1]
```diff
@@ -30,11 +30,12 @@ <h3 class="va-nav-linkslist-heading">{{ group.heading }}</h3>
     {% endif %}
 
     {% if group.links.length > 0 %}
+    {% assign group_heading = group.heading %}
       <ul class="va-nav-linkslist-list">
       {% for link in group.links %}
         {% if link.url != empty %}
           <li>
-            <a href="{{link.url}}">
+            <a href="{{link.url}}" onClick="recordEvent({ event: 'nav-linkslist', 'links-list-header':  '{{ group_heading }}'  });">
```
Comment (cvalarida) [inline src/site/components/merger-crosslinks.html]: Can you use `group.heading` here instead of assigning another variable?

Comment (zurbergram) [reply src/site/components/merger-crosslinks.html]: I tried using `group.heading` but for scoping reasons it wasn't working

File: src/applications/static-pages/school-resources/ScoAnnouncementsWidget.jsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment';
 import { DEFAULT_ANNOUNCEMENT_DAYS_TO_DISPLAY } from './constants/constants';
+import recordEvent from '../../../platform/monitoring/record-event';
```
Comment (shawkey-daniel) [inline src/applications/static-pages/school-resources/ScoAnnouncementsWidget.jsx]: Can't we use the platform alias here?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/static-pages/school-resources/ScoAnnouncementsWidget.jsx (hunk 2 above)
Comment (zurbergram) [reply src/applications/static-pages/school-resources/ScoAnnouncementsWidget.jsx]: fixed

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/site/components/merger-crosslinks.html (hunk 1 above)
Comment (cvalarida) [reply src/site/components/merger-crosslinks.html]: Liquid. :man_facepalming: 
:man_shrugging:

**Review by cvalarida — APPROVED** [review]

**Pull request merged**